src/jira/client/jira-client-audit-log-helper.ts

```diff
@@ -0,0 +1,127 @@
+import { AuditInfo, auditLog } from "../../services/audit-log-service";
+import { isArray, isObject } from "lodash";
+
+const getRepoData = (repoId,request) =>{
+	const repositories = request["repositories"] || null;
+	const acceptedDevinfoEntityData = repositories?.find(({ id })=>id===repoId);
+	return acceptedDevinfoEntityData;
+};
+
+const getAuditInfo = ({
+	githubEntity,
+	repoData,
+	githubEntityType,
+	options
+}) => {
+	const auditInfo: Array<AuditInfo> = [];
+	const createdAt = new Date();
+	githubEntity.map((githubEntityId) => {
+		const repoEntities = repoData?.[githubEntityType];
+		const repoEntity = repoEntities.find(({ id }) => id === githubEntityId);
+		const issueKeys = repoEntity.issueKeys;
+		issueKeys.map((issueKey) => {
+			const obj: AuditInfo = {
+				createdAt,
+				entityId: githubEntityId,
+				entityType: githubEntityType,
+				issueKey,
+				subscriptionId: options?.subscriptionId,
+				source: options?.operationType || "NORMAL",
+				entityAction: options?.entityAction || "null"
+			};
+			if (obj.subscriptionId && obj.entityId) {
+				auditInfo.push(obj);
+			}
+		});
+	});
+	return auditInfo;
+};
+
+export const processBatchedBulkUpdateResp = ({
+	request,
+	response,
+	options,
+	logger
+}): {
+	isSuccess: boolean;
+	auditInfo?: Array<AuditInfo>;
+} => {
+	try {
+		const isSuccess = response?.status === 202;
+		const acceptedDevinfoEntities =
+			response?.data && response?.data?.acceptedDevinfoEntities;
+		const hasAcceptedDevinfoEntities = isObject(acceptedDevinfoEntities) && Object.keys(acceptedDevinfoEntities).length > 0;
+		let auditInfo: Array<AuditInfo> = [];
+		if (isSuccess && hasAcceptedDevinfoEntities) {
+			Object.keys(acceptedDevinfoEntities).forEach(
+				(acceptedDevinfoEntityID) => {
```

Comment:
```suggestion
				(acceptedDevinfoRepoID) => {
```
So that to avoid confusion with `entityID` in the audit log